src/imgui/ImGuiUtils.cc

```diff
@@ -283,6 +284,30 @@ std::string getShortCutForCommand(const HotKey& hotkey, std::string_view command
 	return "";
 }
 
+std::optional<std::string> getKeyChordName(ImGuiKeyChord keyChord)
+{
+	int keyCode = ImGuiKey2SDL(ImGuiKey(keyChord & ~ImGuiMod_Mask_));
+	if (keyCode != SDLK_UNKNOWN) {
+		auto scanCode = SDL_GetScancodeFromKey(keyCode);
+		return strCat((keyChord & ImGuiMod_Ctrl ? "Ctrl+" : ""),
+			(keyChord & ImGuiMod_Shift ? "Shift+" : ""),
+			(keyChord & ImGuiMod_Alt ? "Alt+" : ""),
+			(keyChord & ImGuiMod_Super ? (ImGui::GetIO().ConfigMacOSXBehaviors ? "Cmd+" : "Super+") : ""),
+			SDL_GetScancodeName(scanCode));
+	}
+	return {};
+}
+
+std::optional<ImGuiKeyChord> getKeyChordValue(const std::string_view name)
+{
+	ImGuiKeyChord keyMods = {};
+	keyMods |= StringOp::containsCaseInsensitive(name, "Ctrl+")  ? ImGuiMod_Ctrl  : 0;
+	keyMods |= StringOp::containsCaseInsensitive(name, "Shift+") ? ImGuiMod_Shift : 0;
+	keyMods |= StringOp::containsCaseInsensitive(name, "Alt+")   ? ImGuiMod_Alt   : 0;
```

Comment:
Detail: how general do we need this to be?
Should we only be able to parse strings generated by the `getKeyChordName()` function, or also strings that might have been edited by the user.
* In case of the former it might be cheaper to replace `StringOp::containsCaseInsensitive` with `contains` (case sensitive).
* In case of the latter we might also need to support stuff like `G+Shift`, and then looking for sub-strings like `Shift+` is not sufficient (instead we may need to split on '+').

Personally I think the 1st option should be sufficient.

Reply:
I think `G+Shift` is weird because the `bind` command should enforce a certain order and not even users would consciously write it like this. That's why I preferred to call the attribute something other than "key". So, even if it is editable by the user, mods-before-key should be enough.